src/api/spec/controllers/webui/request_controller_spec.rb

```diff
@@ -49,12 +49,7 @@
     end
 
     context 'if request does not exist' do
-      before do
-        get :show, params: { number: '200000' }
-      end
-
-      it { expect(flash[:error]).to eq("Can't find request 200000") }
-      it { expect(response).to redirect_to(root_url) }
+      it { expect { get :show, params: { number: '200000' } }.to raise_error(ActiveRecord::RecordNotFound) }
```

Comment:
Now that you removed all that custom code, time to remove the superfluous spec. I'm pretty sure `find_by!` is tested in the `ActiveRecord` test suite :)

Reply:
Yep, that's right!